### DISCUSSION TITLE
kata 10 segundo ejercicio ¿es en py o jupiter?

### DISCUSSION THREAD

**Discussion by Ingrzs** [LaunchX-InnovaccionVirtual/CursoIntroPython#217] [Q&A]
Bueno así como dice el titulo, el kata 10 en la parte para crear una función para astronautas, yo cree el archivo en py y en jupiter y pues ejecuta el código pero no pasa nada, no me arroja el resultado de la función del código... Gracias y buenas noches :D

**Comment by YonadabNP** [comment]
Hola! 
Si te refieres a la sección de "Generación de excepciones", todos los ejercicios descritos ahí se ejecutan dentro de un archivo ipynb (Jupiter), o al menos así fue como un servidor pudo ejecutar ese código (recuerda después mandar llamar la función que acabas de crear para poder comprobar los resultados):

![image](https://user-images.githubusercontent.com/99059719/153743328-e19fe7ce-454b-4f22-af75-7f5a2550d8ac.png)

Espero te sirva la información!

Saludos.

**Reply by Ingrzs** [reply]
Tengo una duda, pero en las instrucciones mas arriba antes de crear las funciones, te dice crear un archivo open.py y otro config.py , y luego en la parte mandar a llamarlo con $python config.py en esa parte no logro entenderlo,como lo mando a llamar, yo hice estoy cree un open.py como kata, luego otro config.py como otro kata, luego ejecute el config.py dandole play o donde ejeuctar y si corria todo, tal como muestra error en las carpetas...  , pero todo eso aparecía en modo consola (terminal debajo de vscode)

**Reply by Nukario** [reply]
> Tengo una duda, pero en las instrucciones mas arriba antes de crear las funciones, te dice crear un archivo open.py y otro config.py , y luego en la parte mandar a llamarlo con $python config.py en esa parte no logro entenderlo,como lo mando a llamar, yo hice estoy cree un open.py como kata, luego otro config.py como otro kata, luego ejecute el config.py dandole play o donde ejeuctar y si corria todo, tal como muestra error en las carpetas... , pero todo eso aparecía en modo consola (terminal debajo de vscode)

Esa parte la ejecutas en el CMD pones el escribes "cd" para cambiar la ubicacion y la pones donde tienes los archivos y ahi ejecutas esos codigos para que funcione el "$" no lo pones, eso es para linux, el resto del codigo si lo pones y deberia funcionar
